mithril-client/src/lib.rs

```diff
@@ -62,6 +62,28 @@
 //! #    Ok(())
 //! # }
 //! ```
+//!
+//! ## Optional Features
+//!
+//! The following are a list of [Cargo features](https://doc.rust-lang.org/stable/cargo/reference/manifest.html#the-features-section) that can be
+//! enabled or disabled:
+//!
+//! - **fs**: Enables file system related functionalities.
+//! - **unstable**: Enables experimental or in-development `mithril-client` features that may change.
+//! - **native-tls** *(enabled by default)*: Enables TLS functionality provided by `native-tls`.
```

Comment:
Since they are numerous, adding a clear split for reqwest re-exported features would add clarity:

```suggestion
//! - **unstable**: Enables experimental or in-development `mithril-client` features that may change.
//!
//! To allow fine tuning of the http queries, the following [Reqwest](https://docs.rs/reqwest/latest/reqwest/#optional-features) features are re-exported:
//! - **native-tls** *(enabled by default)*: Enables TLS functionality provided by `native-tls`.
```

Reply:
It's done, thank you for the comment.